.circleci/config.yml

```diff
@@ -25,15 +24,24 @@ references:
     K8S_VERSION: 1.14
     NUM_NODES: 5
 
-  nightly_environment: &nightly_environment
-    DOKS_CLUSTER_NAME: codacy-doks-cluster-release
-    DO_TF_WORKSPACE: release
+  nightly_base_environment: &nightly_base_environment
     RELEASE_NAME: codacy-nightly
     NAMESPACE: codacy-nightly
     NODE_TYPE: s-20vcpu-96gb
-    K8S_VERSION: 1.15
     NUM_NODES: 4
-  
+
+  nightly_environment_1_14: &nightly_environment_1_14
+    <<: *nightly_base_environment
+    DO_TF_WORKSPACE: nightly-1-14
+    DOKS_CLUSTER_NAME: codacy-doks-cluster-nightly-1-14
```

Comment:
We need to be creating/manage the changes of these clusters in this repo and pipeline also

Reply:
How do you suggest we create them on the pipelines? 
Just verify if they do not exist, and if that's the case create them, otherwise continue?

Reply:
Because otherwise we are not using the jobs as part of the pipelines

Reply:
`Just verify if they do not exist, and if that's the case create them, otherwise continue?` -> yes

Reply:
and change if anything is different... 
basically, make sure that the infrastructure matches the configuration in the repo